src/app/components/hearing/order-confirmation/order-confirmation.component.html

```diff
@@ -16,7 +16,7 @@ <h3 class="govuk-heading-s">Defendant(s)</h3>
     <p class="govuk-body">{{ case.defendants }}</p>
     <h2 class="govuk-heading-m">Audio details</h2>
     <h1 class="govuk-heading-s">Hearing date</h1>
-    <h1 class="govuk-body">{{ hearing?.date }}</h1>
+    <h1 class="govuk-body">{{ hearing?.date | date: 'dd/MM/yyyy' }}</h1>
```

Comment:
this should be
`<h1 class="govuk-body">{{ hearing?.date | date: 'd MMM y' }}</h1>`